extensions/obsidian/src/index.tsx

```diff
@@ -66,6 +72,20 @@ function getFiles(vaultPath: string) {
   return files;
 }
 
+function prefVaults() {
+  const pref: Preferences = getPreferenceValues();
+  const vaultString = pref.vaultPath
+  if (vaultString) {
+    const vaults = vaultString.split(",");
+    for (let i = 0; i < vaults.length; i++) {
+      vaults[i] = vaults[i].trim();
+    }
+    return vaults;
```

Comment:
Just a stylistic suggestion
```suggestion
    return vaultString.split(",").map(vault => vault.trim()).filter(vault => !!vault);
```

This would also make sure that no empty vault are left (which might lead to unexpected behaviours down the line)

Reply:
Thank you. I implemented this in `openvault.tsx` too.